### ISSUE TITLE
class 'AttributeError: module 'base64' has no attribute 'encodestring'

### ISSUE THREAD

**Issue by jamieburchell** [s3tools/s3cmd#1161]
```
Problem: <class 'AttributeError: module 'base64' has no attribute 'encodestring'
S3cmd:   2.1.0
python:   3.9.0 (default, Nov 30 2020, 13:01:15) 
[GCC 8.4.0]
environment LANG=en_US.UTF-8

Traceback (most recent call last):
  File "/opt/bin/s3cmd", line 3121, in <module>
    rc = main()
  File "/opt/bin/s3cmd", line 3030, in main
    rc = cmd_func(args)
  File "/opt/bin/s3cmd", line 1898, in cmd_sync
    return cmd_sync_local2remote(args)
  File "/opt/bin/s3cmd", line 1871, in cmd_sync_local2remote
    ret = _single_process(args[:-1])
  File "/opt/bin/s3cmd", line 1625, in _single_process
    return _child(destination_base, source_args)
  File "/opt/bin/s3cmd", line 1809, in _child
    subcmd_batch_del(remote_list = remote_list)
  File "/opt/bin/s3cmd", line 743, in subcmd_batch_del
    _batch_del(remote_list)
  File "/opt/bin/s3cmd", line 719, in _batch_del
    response = s3.object_batch_delete(to_delete)
  File "/opt/lib/python3.9/site-packages/S3/S3.py", line 732, in object_batch_delete
    self.object_batch_delete_uri_strs(uris)
  File "/opt/lib/python3.9/site-packages/S3/S3.py", line 757, in object_batch_delete_uri_strs
    headers = SortedDict({'content-md5': compute_content_md5(request_body),
  File "/opt/lib/python3.9/site-packages/S3/S3.py", line 1831, in compute_content_md5
    base64md5 = base64.encodestring(m.digest())
AttributeError: module 'base64' has no attribute 'encodestring'

!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
    An unexpected error has occurred.
  Please try reproducing the error using
  the latest s3cmd code from the git master
  branch found at:
    https://github.com/s3tools/s3cmd
  and have a look at the known issues list:
    https://github.com/s3tools/s3cmd/wiki/Common-known-issues-and-their-solutions
  If the error persists, please report the
  above lines (removing any private
  info as necessary) to:
   s3tools-bugs@lists.sourceforge.net
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

**Comment by jamieburchell** [comment]
I see this was resolved by changes in master

**Comment by alanorth** [comment]
Fixed by: https://github.com/s3tools/s3cmd/commit/44cc1a996658eb6d150973018fece5b6d6e1f5bf

**Comment by ttcremers** [comment]
> Fixed by: [44cc1a9](https://github.com/s3tools/s3cmd/commit/44cc1a996658eb6d150973018fece5b6d6e1f5bf)

In which version will this be released? Thanks

**Comment by sorenwacker** [comment]
I still got this error. Installed s3cmd with conda.
